test/extended/operators/check_crd_status.go

```diff
@@ -98,6 +98,7 @@ func checkStatusInSchema(crdItemList []apiextensionsv1.CustomResourceDefinition)
 		"imagecontentpolicies.config.openshift.io",
 		"imagecontentsourcepolicies.operator.openshift.io",
 		"machineconfigs.machineconfiguration.openshift.io",
+		"pinnedimagesets.machineconfiguration.openshift.io",
```

Comment:
/hold

don't do this.  You could add checking to ensure no status exists, but this opens up to severe errors around permissioning and semantics we cannot fix.

Reply:
the original goal of the testing was

> they aren't required to have status. Some objects don't. But if they have status in the schema, they must have a status subresource

https://github.com/openshift/origin/pull/26954#discussion_r860937576

if the implementation got confused, that can be resolved.  But if a status subelement exists, it *must* have a subresource for it.